actors/miner/src/lib.rs

```diff
@@ -1287,9 +1288,12 @@ impl Actor {
                     // Skip checking if CID is defined because it cannot be so in Rust
 
                     new_sector_info.deal_ids = with_details.update.deals.clone();
-                    new_sector_info.activation = rt.curr_epoch();
 
-                    let duration = new_sector_info.expiration - new_sector_info.activation;
+                    if !with_details.deal_spaces.verified_deal_space.is_zero() {
+                        new_sector_info.power_base_epoch = rt.curr_epoch();
```

Comment:
Let's add a comment explaining why this is important?

Reply:
Ok, actually... I think we should either:

1. Always update the power base epoch.
2. _Not_ update the pledge, etc.

Honestly, I'd rather just _always_ update the power base epoch. Does anyone do non-verified snap deals anyways?

Reply:
i'm gonna just always set the PBE, i think it makes more sense